controllers/apps/apimanager_status_reconciler.go

```diff
@@ -0,0 +1,234 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	appsv1alpha1 "github.com/3scale/3scale-operator/apis/apps/v1alpha1"
+	"github.com/3scale/3scale-operator/pkg/3scale/amp/component"
+	"github.com/3scale/3scale-operator/pkg/common"
+	"github.com/3scale/3scale-operator/pkg/helper"
+	"github.com/3scale/3scale-operator/pkg/reconcilers"
+	"github.com/RHsyseng/operator-utils/pkg/olm"
+	"github.com/go-logr/logr"
+	appsv1 "github.com/openshift/api/apps/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type APIManagerStatusReconciler struct {
+	*reconcilers.BaseReconciler
+	apimanagerResource *appsv1alpha1.APIManager
+	syncError          error
```

Comment:
Added to be prepared but not currently used

Reply:
Finally I've removed this due to not being used. We can always add it in the future in case is needed.